rdkit/Chem/Pharm3D/Pharmacophore.py

```diff
@@ -58,74 +59,72 @@ def getFeature(self, i):
     return self._feats[i]
 
   def getUpperBound(self, i, j):
-    if (i > j):
+    if i > j:
       j, i = i, j
     return self._boundsMat[i, j]
 
   def getLowerBound(self, i, j):
-    if (j > i):
+    if j > i:
       j, i = i, j
     return self._boundsMat[i, j]
 
   def _checkBounds(self, i, j):
     " raises ValueError on failure "
     nf = len(self._feats)
-    if (i < 0) or (i >= nf):
-      raise ValueError("Index out of bound")
-    if (j < 0) or (j >= nf):
-      raise ValueError("Index out of bound")
-    return True
+    if (0 <= i < nf) and (0 <= j < nf):
+      return True
+    raise ValueError("Index out of bound")
 
   def setUpperBound(self, i, j, val, checkBounds=False):
-    if (checkBounds):
+    if checkBounds:
       self._checkBounds(i, j)
-    if (i > j):
+    if i > j:
       j, i = i, j
     self._boundsMat[i, j] = val
 
   def setLowerBound(self, i, j, val, checkBounds=False):
-    if (checkBounds):
+    if checkBounds:
       self._checkBounds(i, j)
-    if (j > i):
+    if j > i:
       j, i = i, j
     self._boundsMat[i, j] = val
 
   def getUpperBound2D(self, i, j):
-    if (i > j):
+    if i > j:
       j, i = i, j
     return self._boundsMat2D[i, j]
 
   def getLowerBound2D(self, i, j):
-    if (j > i):
+    if j > i:
```

Comment:
(sigh) all of these C artifacts in this old code...
At least the semicolons are all gone